scripts/o3de/tests/test_repo.py

```diff
@@ -65,6 +65,130 @@
 }
 '''
 
+#This holds the repo 2.0.0 schema, which houses all relevant objects in file
+TEST_O3DE_REPOC_FILENAME = '3b14717bafd5a3bd768d3d0791a44998c3bd0fb2bfa1a7e2ee8bb1a39b04d631.json'
+TEST_O3DE_REPOC_JSON_PAYLOAD = '''
+{
+	"repo_name": "testgem3",
```

Comment:
looks like this file has some tabs and spaces mixed, should be converted to spaces